libs/pages/application/src/lib/feature/page-settings-dockerfile-feature/__snapshots__/page-settings-dockerfile-feature.spec.tsx.snap

```diff
@@ -0,0 +1,7 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`PageSettingsDockerfileFeature should match snapshot 1`] = `
+<body>
+  <div />
+</body>
+`;
```

Comment:
Missing something to generate snapshot ?

Reply:
https://github.com/Qovery/console/pull/1433/commits/b6deb42b3b9817f053b94d9878abb355eb91b189